docker-compose.yml

```diff
@@ -0,0 +1,20 @@
+version: "3.9"  # optional since v1.27.0
+services:
+  bot:
```

Comment:
We should use a [`depends_on`](https://docs.docker.com/compose/compose-file/compose-file-v3/#depends_on) option for the bot to ensure Redis spins up before the bot starts.

Reply:
`links`  already implies `depends_on`, but if we're removing `links` then yes, this would make sense.

> Links also express dependency between services in the same way as depends_on, so they determine the order of service startup.